lib/recurly/coupon.php

```diff
@@ -1,5 +1,36 @@
 <?php
 
+/**
+ * Class Recurly_Coupon
+ * @property Recurly_Stub $redemptions The URL of Recurly_CouponRedemptions for this coupon.
+ * @property int $id The unique integer identifier of the coupon.
+ * @property string $coupon_code Unique code to identify and redeem the coupon. This code may only contain the following characters: [a-z A-Z 0-9 + - _ ]. Max of 50 characters.
+ * @property string $name Coupon name.
+ * @property string $description Description of the coupon on the hosted payment pages.
+ * @property string $discount_type The type of discount. Allowed values: [percent, dollars, free_trial].
+ * @property int $discount_in_cents Mapping of discount amounts by currency if discount_type is dollars. Max 10000000.
+ * @property int $discount_percent Discount percentage if discount_type is percent. Example coupon->discount_percent = 10; // 10%
```

Comment:
If we're doing inline examples I think I'd expect some kind of quoting around it, perhaps backticks?